### PR TITLE
Front signin sign up

### PR DESCRIPTION
**Pull request by kirouackp0**
### Ticket Link
(Found on Trello in your ticket description)
https://trello.com/c/sNHXVUTS

### User Story
(Linked to on Trello in your ticket description)
https://trello.com/c/JMOoUKw2

### What does this PR do?
(Your answer here.)
provides tentative front end for user registration page (CSS still needs work and JS to be added)

### Where should the reviewer start?
(Your answer here.)
Perhaps having a look at manifest.js. The rest is very straightforward CSS.

### How should this be manually tested?
N/A

### Any background context you want to provide?
If time later - will add animation and change color of filled checkbox.

### Screenshots (if appropriate)
N.A

### Are there any breaking changes?
The manifest.js was updated with new stylesheet as it was not working without. this causes no problems locally but should be flagged.

### CODE REVIEW

**Comment by Creiwry** [comment]
If this is just a tentative version, no need to do a PR yet. As long as you haven't changed layout massively so the others need to know, it's ok to keep committing to your branch per feature and then update PR with new information once the code is more complete.

**Comment by kirouackp0** [comment]
New commit - ready to merge

File: Gemfile [hunk 1]
```diff
@@ -86,3 +86,5 @@ gem "dockerfile-rails", ">= 1.5", :group => :development
 gem "sentry-ruby", "~> 5.10"
 
 gem "sentry-rails", "~> 5.10"
+
+gem "tailwindcss-rails", "~> 2.0"
```
Comment (greg-Da) [inline Gemfile:90]: On a supprimé tailwind de l app

File: Procfile.dev
Comment (greg-Da) [inline Procfile.dev:1]: Il faut supprimer ce fichier

**Review by greg-Da — CHANGES_REQUESTED** [review]
Est ce que tu peux faire un `git pull origin main` et gerer les conflits je peux pas le faire puisque tu es sur un fork

File: Procfile.dev
Comment (kirouackp0) [reply Procfile.dev:1]: Fait

File: Gemfile (hunk 1 above)
Comment (kirouackp0) [reply Gemfile:90]: Supprimé

**Pull request merged**